### PR TITLE
Review first version before release to the master

### PR DESCRIPTION
**Pull request by ruchidesai**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+modification, are permitted provided that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
+
+* Redistributions in binary form must reproduce the above copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+
+* Neither the name of nipype-pbs-workflows nor the names of its
```
Comment (pbchase) [inline LICENSE]: This should reference "University of Florida" instead of "nipype-pbs-workflows"

File: README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+## nipype-workflows
+
+This repository consists of neuroimaging workflows writtten in nipype with a focus on the Portable Batch System (PBS) job scheduler.
```
Comment (pbchase) [inline README.md]: written

File: test [hunk 3]
```diff
@@ -0,0 +1 @@
+test
```
Comment (pbchase) [inline test]: Does this file serve a purpose?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+It provides tools and documentation for BMC users using the services of UF's Researchcomputing (RC) group.
```
Comment (pbchase) [inline README.md]: How about this:

```
It provides tools and documentation for neuroscientists to using the services of UF's Research Computing (RC) group.
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+### Installation
+
+In order to run the code provided in this repository the following software need to be avaliable:
```
Comment (pbchase) [inline README.md]: available

**Comment by pbchase** [comment]
Ruchi,  

This looks great.  If you can make the changes I described this morning I'd love to merge this today.

Thanks,
Philip

**Comment by ruchidesai** [comment]
Hi Philip,

I have made the changes suggested by you.

Thanks,
Ruchi.

**Pull request merged**